src/app/modules/main/profile_section/wallet/accounts/module.nim

```diff
@@ -104,7 +105,11 @@ method load*(self: Module) =
       return
     self.refreshWalletAccounts()
 
+  self.events.on(SIGNAL_INCLUDE_WATCH_ONLY_ACCOUNTS_UPDATED) do(e: Args):
+    self.view.setIncludeWatchOnlyAccount(self.controller.isIncludeWatchOnlyAccount())
+
   self.controller.init()
+  self.view.setIncludeWatchOnlyAccount(self.controller.isIncludeWatchOnlyAccount())
```

Comment:
Logical order should be that you first need to load the view (line `113`) and then line `112`.